src/platform/Darwin/DnssdImpl.h

```diff
@@ -81,13 +81,14 @@ class MdnsContexts
      *                      of the service name, the service protocol, and the PTR records.
      *                      Example:
      *                        _matterc._udp,_V65521,_S15,_L3840,_CM
-     *                        _matter._tcp,_I4CEEAD044CC35B63
```

Comment:
hmm. So this comment used to be true for both commissionable and operational advertisements in https://github.com/project-chip/connectedhomeip/blame/e4df5595f3b99f69f3ca272c99594f391d0c6321/src/platform/Darwin/DnssdImpl.cpp#L582

It seems that it is no longer true since a while. Can you also update the comment for the commissionable type ?

Reply:
Looks like I was just wrong for the operational case: for operational we have an "_I" subtype that lists the compressed fabric ID.  Will just undo this comment change.